features/registering_assets.feature

```diff
@@ -12,8 +12,8 @@ Feature: Registering Assets
 
   Scenario: Viewing default asset files
     When I am on the index page for posts
-    Then I should see the css file "active_admin.css"
-    Then I should see the js file "active_admin.js"
+    Then I should see the css file "active_admin"
```

Comment:
Since our rough conclusion was that `webpacker` is opt-in and we're not using it, I have the feeling that the old setup might actually work. Should we double check this?